src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -149,6 +149,8 @@ public ConnectionManagerSettings(NodeSettings nodeSettings)
 
             this.BanTimeSeconds = config.GetOrDefault<int>("bantime", nodeSettings.Network.IsTest() ? DefaultMisbehavingBantimeSecondsTestnet : DefaultMisbehavingBantimeSeconds, this.logger);
             this.MaxOutboundConnections = config.GetOrDefault<int>("maxoutboundconnections", nodeSettings.Network.DefaultMaxOutboundConnections, this.logger);
+            if (this.MaxOutboundConnections <= 0)
+                throw new ConfigurationException("The 'maxoutboundconnections' must be greater than zero.");
             this.MaxInboundConnections = config.GetOrDefault<int>("maxinboundconnections", nodeSettings.Network.DefaultMaxInboundConnections, this.logger);
```

Comment:
The name of the issue implies that this setting should also be checked? Possibly not needed as it may be desirable to optionally inhibit inbound connections entirely. Less than zero would still be invalid though.

Reply:
Done.